gqcpy/src/QCModel/CI/LinearExpansion_bindings.cpp

```diff
@@ -181,18 +170,24 @@ void bindQCModelCILinearExpansionDensityMatrixInterface(Class& py_class) {
 void bindLinearExpansions(py::module& module) {
 
     // Define the python class related to a linear expansion of a Seniority Zero ONV basis and expose the necessary interfaces.
-    py::class_<LinearExpansion<SeniorityZeroONVBasis>> py_LinearExpansion_SeniorityZero {module, "LinearExpansion_SeniorityZero", "The linear expansion (configuration interaction) wave function model in a seniority-zero ONV basis."};
+    py::class_<LinearExpansion<double, SeniorityZeroONVBasis>> py_LinearExpansion_SeniorityZero {module, "LinearExpansion_SeniorityZero", "The linear expansion (configuration interaction) wave function model in a seniority-zero ONV basis."};
 
     // Expose the bindings exclusive to this type of linear expansion
     py_LinearExpansion_SeniorityZero
 
         /*
          * MARK: Entropy
          */
+        .def(
+            "calculateShannonEntropy",
+            [](const LinearExpansion<double, SeniorityZeroONVBasis>& linear_expansion) {
+                return linear_expansion.calculateShannonEntropy();
+            },
+            "Return the Shannon entropy (information content) of the wave function.")
```

Comment:
```suggestion
```
The calculateShannonEntropy API is part of the interface I believe, so you can delete this here.

Reply:
As we discussed, I had to move this API out of the 'binding interface' since it cannot be enabled (yet) for complex-valued coefficients.